### ISSUE TITLE
If getText() is called for an "paste" event it only returns a space

### ISSUE THREAD

**Issue by mactr0n** [mervick/emojionearea#11]
This returns " ":
```
       emojioneArea[0].emojioneArea.on('keyup paste emojibtn.click change',
                function () {
                    console.log(emojioneArea[0].emojioneArea.getText());
                });
```
This returns " text":
```
  emojioneArea[0].emojioneArea.on('keyup paste emojibtn.click change',
                function () {
                    setTimeout(function() {
                        scope.$apply(function () {
                            console.log(emojioneArea[0].emojioneArea.getText());
                        });
                    }, 300);
                });
```

1. Why is there an delay?
2. Why is there a space?

**Comment by mactr0n** [comment]
Fount the reason for the space in front of each pasted text:
```
.on("@paste", function(element) {
  stayFocused = true;
  pasteHtmlAtCaret('<span> </span>');
```

**Comment by mactr0n** [comment]
Whats the reason for the timeout in the paste event?

**Comment by mervick** [comment]
Fixed in [v2.1.1](https://github.com/mervick/emojionearea/releases/tag/v2.1.1), try it now
```js
$("selector").emojioneArea({
    // options
});
$("selector").data("emojioneArea").on("paste", function(editor, text, html) {
    console.log([editor, text, html, this.getText()]);
});
```
